index.js

```diff
@@ -1,102 +1,106 @@
 /**
- * Винести змінні які ти мутуєш в окремий об'єкт як ключі
- * Пройтись по змінних і замінити let на const у місцях де це можливо
- * Змінні в camelCase
- * DRY principle: накнайменше дублікацій коду
- * Переписати проміси на async/await
+ * Винести змінні які ти мутуєш в окремий об'єкт як ключі +
+ * Пройтись по змінних і замінити let на const у місцях де це можливо +
+ * Змінні в camelCase +
+ * DRY principle: накнайменше дублікацій коду +
+ * Переписати проміси на async/await +
  * */
 
 
 (function (){
   const pagination = {
-
+    page: 1,
+    perPage: 2,
+    total: document.getElementById('total'),
+    totalCount: 0,
+    pageNumberSpan: document.getElementById('page-number'),
+    pageNumber: 1,
+    btnNext: document.getElementById('btn_next'),
+    btnPrev: document.getElementById('btn_prev'),
+    url: 'https://reqres.in/api/users?'
```

Comment:
It's better to move the `url` property to a separate variable (F.E `API_URL`). The same for the properties with DOM elements (`total`, `pageNumberSpan`, `btnNext`, `btnPrev`). So, let's create separated variables with them. In the `pagination` object should be only data that can be changed from the UI: `page`, `perPage`, `totalCount` and `pageNumber`);